packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -296,8 +296,15 @@ export default {
     };
 
     const getDomainNode = (domain: GridGateway): string => {
-      const getIP = (url: string) => new URL(url).hostname;
-      const IP = getIP(domain.backends[0]);
+      const extractIP = (input: string) => {
+        try {
+          return new URL(input).hostname;
+        } catch {
+          return input.split(":")[0];
+        }
+      };
+
+      const IP = extractIP(domain.backends[0]);
```

Comment:
can we stick into 1 solution either prefix url with `http://` so URL work with ip or use split with both cases so we don't get confuse later